.github/workflows/build.yml

```diff
@@ -70,8 +70,6 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - macOS-10.15
-          - windows-2019
```

Comment:
@mosyp this will remove the azure tests for those platforms as well, let's further break up azure and aws tests into different jobs for now so we don't skip azure for those platforms.

Reply:
yeap, makes sense

Reply:
btw, does it actually make sense to run azure on these platforms, since it is cloud based?